code/src/reducers/todos.js

```diff
@@ -0,0 +1,60 @@
+import { createSlice } from '@reduxjs/toolkit';
+import uniqid from 'uniqid';
+
+const todos = createSlice({
+    name: 'todos',
+    initialState: {
+        items: [
+            {
+                id: uniqid(),
+                text:'Submit project',
+                isComplete: false,
+            },
+            {
+                id: uniqid(),
+                text:'Clean the house',
+                isComplete: false,
+            },
+            {
+                id: uniqid(),
+                text:'Do the dishes',
+                isComplete: false,
+            } 
+        ],
+    },
+    //The actions match the names of the reducers
+    reducers: {
+        removeItem: (state, action) => {
+            const itemId = action.payload;
+            const filteredList = state.items.filter((item => item.id !== itemId));
+            state.items = filteredList;
+
+        },
+        changeItem: (state, action) => {
```

Comment:
Naming is hard so just suggestions here:
`changeCompleteStatus`, `toggleComplete`, `toggleIsComplete`